src/satbias/GsiSatBiasReader.cpp

```diff
@@ -0,0 +1,111 @@
+/*
+ * (C) Copyright 2021 UCAR
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+#include "GsiSatBiasReader.h"
+
+#include <string>
+#include <vector>
+#include <fstream>
+
+#include <Eigen/Dense>
+
+constexpr size_t npredictors = 12;
```

Comment:
should this be in the header?

Reply:
I've decided to keep it here, since it's only used in this file.
But it probably should be in the header; there should be a check whether number of predictors specified in yaml is the same as this one; I'll add that.

Reply:
I've moved this to the header, and added a check on number of predictors.